members/C001056.yaml

```diff
@@ -0,0 +1,96 @@
+bioguide: C001056
+contact_form:
+  method: POST
```

Comment:
we should go ahead and stop adding `method` to new yamls, as it isn't used and is just clutter

Reply:
deleted it